Makefile

```diff
@@ -615,7 +622,7 @@ realclean: clean java_clean
 	@$(RM) -f *.bin debian/changelog*
 
 ### additionally removes generated files not in the repository tree
-distclean: realclean clean_genRules
+distclean: clean_stack realclean clean_genRules
```

Comment:
Yepp, docs are very shallow and often really vague - they probably embraced the "try and error" * too much ;-)

IIRC, Christian used pbuilder, debuild and possibly other stuff in utils/debian/ (which is dead meat since last major build system update 1+ year ago), which is very inefficient, introduces a lot of overhead (ein vermotteter Flickenteppich aus paranoidem/kranken Zeux ;-) )  and steals one a lot of time ...

Therefore I decided to use source format 2.0 and to create the required description as well as changes file incl. src archive for each release directly by myself using the "lower level" tools directly (dpkg-genchanges, dpkg-source + sha1, sha256, md5, gpg2, tar, sftp) in one row.
But perhaps you find a different way to do it with the deb[u]il[d] tool.

Wrt. download or not: as said, I'm not sure, maybe this have changed. At least it is not documented, so just try if you need it.

Wrt. "all have to use the same": well, good software usually uses macros/defines or similar things to honor the users environment/wishes and to work around platform/release specific issues, like var.mk does. When you think, that stack solves your problems magically, then I guess you'll hit sooner or later a wall, which requires to think over it again ...

BTW: I consider all the "I re-invent the wheel and do my own packaging stuff/bypassing the system in use" very dangerous - for development it is ok wrt. being able to work on the cutting edge, but the distributed stuff, which people aka end users will actually use on their systems should smoothly integrate into this environment and use system packages/software as much as it can.